target/linux/bmips/image/bcm63268.mk

```diff
@@ -1,5 +1,24 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 
+define Device/actiontec_t1200h
+  $(Device/bcm63xx-nand)
+  DEVICE_VENDOR := Actiontec
+  DEVICE_MODEL := T1200H
+  CHIP_ID := 63268
+  SOC := bcm63168
+  CFE_RAM_FILE := actiontec,t1200h/cferam.000
```

Comment:
Now that https://github.com/openwrt/bcm63xx-cfe/pull/7 has been merged we can also bump bcm63xx-cfe package to the latest version on a separate commit in order to have **actiontec,t1200h/cferam.000**.